content/supertokens-vs-clerk/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Clerk Vs Supertokens: Custom Vs Ready-To-Use Access Protocol"
+date: "2024-06-01"
+description: "Explore the differences between SuperTokens and Clerk to determine the best user access management solution for your project. Understand their core functionalities, key differences, and pricing to make an informed decision."
+cover: "supertokens-vs-clerk.png"
+category: "programming"
+author: "Darko Bozhinovski"
+---
+
+
+## Introduction
+
+Developers often face a dilemma regarding user access management: Should they invest time and resources in building a custom solution or opt for a ready-to-use tool? This decision impacts both the user experience and the development workflow (and, by extension, the developer experience).
+
+This article will explore SuperTokens and Clerk. By studying their core functionalities, key differences, and pricing structures, we aim to help you make an informed decision that aligns with your project's unique needs.
+
+## Clerk vs. SuperTokens: Two Different Paths To User Access Management
+
+Authentication solutions come in various shapes and sizes, each tailored to different needs and use cases. SuperTokens and Clerk represent two approaches to user access management: customizability vs. out-of-the-box functionality.
+
+### SuperTokens Core Functionalities
+
+SuperTokens is an open-source authentication solution that prides itself on flexibility and customizability. It offers a managed service for hassle-free integration and a self-hosted option for complete control over user data. Key features include:
+
+- Modular Architecture: SuperTokens' modular setup allows developers to pick and choose the functionalities they need, whether email-password login, social logins, or passwordless authentication.
+
+- Customizable Auth Flows: With the override feature, you can tailor authentication flows to your needs on both the front and backend.
+
+- Pre-built UI Components: SuperTokens provides pre-built UI components that can be seamlessly embedded into your existing frontend.
+
+- Session Management: Out-of-the-box support for secure, cookie-based session management.
+
+- User Management Dashboard: SuperTokens includes a user management dashboard, making it easier to manage users and their sessions.
+
+- Social Login Integration: Supports multiple social login providers out of the box.
+
+- Open Source: All SDKs and the core service are open source, encouraging transparency and community contributions.
```

Comment:
we have mfa as well!!! please add that.. dont downplay supertokens unnecessarily